mailpoet/assets/js/src/logs/list.tsx

```diff
@@ -28,7 +28,7 @@ type MessageProps = {
 
 function Message({ message, editing }: MessageProps):JSX.Element {
   if (!editing) {
-    return (<>{`${message.substr(0, 150)}…`}</>);
+    return (<div>{`${message.substr(0, 150)}…`}</div>);
```

Comment:
This changes the DOM... could we simply do this?
```ts
return `${message.substr(0, 150)}…`;
```

Reply:
I agree that changing the DOM is not a perfect solution. I checked, there is no visual change though. I hesitate because of this [playground](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4AoczAVwDsNgJa4AVJAZxgAoBKOALjgApAMoANAHQBRADZIQSWvADelOOrhEY1KM3wwOMMuQC+lKnQZM4mCBF5xV5DZqTbdcLgB4wAPi9snHAA9P7BfjwUZuRIAB6QsHAAJkiYKNQyWHakQA).

Reply:
This would be solved using `allowExpressions` as well (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-useless-fragment.md#allowexpressions).